docs/content/overview.md

```diff
@@ -40,7 +40,7 @@ Here is an overview of the major features of each cloud storage system.
 | Memory                       | MD5               | R/W     | No               | No              | -         | -        |
 | Microsoft Azure Blob Storage | MD5               | R/W     | No               | No              | R/W       | -        |
 | Microsoft Azure Files Storage | MD5              | R/W     | Yes              | No              | R/W       | -        |
-| Microsoft OneDrive           | QuickXorHash ⁵    | R/W     | Yes              | No              | R         | -        |
```

Comment:
I don't know if you saw on the latest push I added a `D` flag indicating ModTime and Metadata work on directories. So I think OneDrive's should be `DR/W` for modtime and `DRW` for Metadata

Reply:
Will make that change!